pydicom/data/data_manager.py

```diff
@@ -40,6 +55,20 @@ def get_files(base, pattern):
             if len(filename_filter):
                 files.append(filename_filter[0])
 
+    # To preserve backwards compatibility filter the downloaded files
+    # as if they are stored within DATA_ROOT/test_files/*.dcm
+    dummy_online_file_path_map = online_test_file_dummy_paths()
+    dummy_online_file_path_filtered = fnmatch.filter(
+        dummy_online_file_path_map.keys(), pattern)
+    download_names = [str(dummy_online_file_path_map[dummy_path])
+                      for dummy_path in dummy_online_file_path_filtered]
+
+    real_online_file_paths = []
+    for filename in download_names:
```

Comment:
How about `downloaded_file_paths` here -- since no longer online?

Reply:
Weirdly, at this point, they may not already be downloaded. The download would actually happen at:

https://github.com/SimonBiggs/pydicom/blob/153e65bddb5bd3f8026aa7c7973f3b09a8671b6f/pydicom/data/data_manager.py#L69

Reply:
Maybe, the variable name could be `online_filename`?

Reply:
> at this point, they may not already be downloaded

Right, I was thinking of the line with the `data_path_with_download`, which is actually the `real_online_file_paths`, so no problem, the naming is all okay.